README.md

```diff
@@ -19,3 +20,28 @@ gem build workos.gemspec
 ### Requirements
 
 - Ruby 2.3+.
+
+# Release Notes
+
+### December 27, 2019
+
+- Removed the `redirect_uri` parameter from the `WorkOS::SSO.profile` function. Migrating existing code just requires removing the existing parameter:
+
+  ```ruby
+  # < December 27, 2019
```

Comment:
Not sure how versioning works w/ Ruby. I'm sure there is a more idiomatic way, but not sure where the current version is located in the repo.

Reply:
couple of moving parts with the versioning

version is defined here: https://github.com/workos-inc/workos-ruby/blob/master/lib/workos/version.rb

I think technically this would be a breaking minor change, so we'd go to `v0.1.0` with this?

but since we have folks installing the gem via github, we need to wait to merge this until we can get in touch with our customers.

They could update to 0.0.2 from ruby gems (releasing that now) or we could have them implement these changes and continue installing from github or push a new version to rubygems. a little messy since we are touch and go with a first customer. I'm gonna link them to this PR

Reply:
Sweet. I'll go ahead and bump the version in that file and update the readme. We can hold off on releasing this version till everything is ready to go.

Reply:
we could maybe also start using github releases for changelog?

alos, you'll want to update yard docs here, theres a bash script in bin/docs that reads the comment style documentation and creates the files for https://workos-inc.github.io/workos-ruby/